assets/style/main.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 5px 8px;
+}
+
+body {
+  background-color: white;
+  font-family: "Poppins", Helvetica, Arial, sans-serif;
+  font-weight: normal;
+}
+
+.section-content {
+  margin-left: 20px;
+  margin-right: 30px;
+}
+
+.common-font-family {
+  font-family: "Poppins", Helvetica, Arial, sans-serif;
+  font-style: normal;
+}
+
+main {
+  background-image: url("../images/myportfoliobgi.png");
+  height: 100vh;
+  width: 375px;
```

Comment:
- [OPTIONAL] I think it would be better to use % and vw as units of width to make your project responsive :recycle: